addons/crm/models/crm_lead.py

```diff
@@ -1957,9 +1957,9 @@ def _pls_get_naive_bayes_probabilities(self, batch_mode=False):
                 if field == 'stage_id' and value in won_stage_ids:
                     won_leads.add(lead_id)
                 leads_fields.add(field)
-
+        leads_fields = sorted(leads_fields)
```

Comment:
Why is it necessary ? I don't see where queries would be impacted by the ordering ?